lib/rules/max-line-length/__tests__/index.js

```diff
@@ -238,7 +238,7 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
-  config: [ 20, { ignorePattern: "/https?:\/\/[0-9,a-z]*.*/" } ],
+  config: [ 20, { ignorePattern: "/https?://[0-9,a-z]*.*/" } ],
```

Comment:
> Per the rule docs though this should be a regex though should it not?

I meant that it's a regular expression _within_ a string e.g. `"/regex/"`, rather than a regular expression itself e.g. `/regex/`. To be honest, all this regex handling and escaping, especially when they're as strings within JSON configuration objects, is a bit beyond me!

The tests are passing though, so does that mean we're OK? Or should we continue to dig deeper?